Modules/Core/Common/include/itkMacro.h

```diff
@@ -485,6 +485,15 @@ OutputWindowDisplayGenericOutputText(const char *);
 
 extern ITKCommon_EXPORT void
 OutputWindowDisplayDebugText(const char *);
+
+/** The default tolerance when comparing the geometry of two images
+ *
+ * The value was chosen based on precisions of file formats such as DICOM,
+ * to enable interoperability with images saved to file formats with higher precision.
+ */
+const double itkDefaultCoordinateTolerance = 1e-6;
+const double itkDefaultDirectionTolerance = 1e-6;
+
```

Comment:
Thanks @blowekamp I very much appreciate having named constants for such "magic numbers". I'm not entirely sure about having them in "itkMacro.h" though, as they obviously aren't really `#define` macro's. Maybe have them in a separate file, "itkDefaultConstants.h" ? Or maybe just place them in "itkImageBase.h"?

By the way, I would rather drop the "itk" prefix from the identifiers, as they are already nicely defined inside the `itk` namespace.

Reply:
I think that Niels is right. These constants feel as they belong in ImageBase, not here.